src/applications/personalization/profile/mocks/feature-toggles/index.js

```diff
@@ -14,6 +14,7 @@ const generateFeatureToggles = (toggles = {}) => {
     profileShowPaymentsNotificationSetting = false,
     profileBlockForFiduciaryDeceasedOrIncompetent = false,
     profileUseSecurityProcessList = true,
+    profileUseInfoCard = false,
```

Comment:
To test this work, you can set the profileUseInfoCard to true and load the Direct Deposit page.